cli/cmd/env/show.go

```diff
@@ -62,6 +69,33 @@ func ShowCmd(ch *cmdutil.Helper) *cobra.Command {
 	return showCmd
 }
 
+func formatEnvVar(name, value string) string {
+	return fmt.Sprintf("%s=%s", name, value)
+}
```

Comment:
If `value` contains a newline, this might throw off the formatting. Newlines could happen e.g. for GCS credentials where we store the JSON payload from the service account `.json` file that the user provides.

Maybe it's as simple as:
```suggestion
func formatEnvVar(name, value string) string {
	return fmt.Sprintf("%s=%q", name, value)
}
```

(unless you want to be fancy and only add quotes conditionally when the value has non-standard characters.)